
Property changes on: .
___________________________________________________________________
Added: svn:externals
   + specialpurpose/appserver/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/appserver
specialpurpose/assetmaint/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/assetmaint
specialpurpose/birt/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/birt
specialpurpose/cmssite/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/cmssite
specialpurpose/ebay/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/ebay
specialpurpose/ebaystore/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/ebaystore
specialpurpose/example/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/example
specialpurpose/exampleext/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/exampleext
specialpurpose/googlebase/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/googlebase
specialpurpose/googlecheckout/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/googlecheckout
specialpurpose/hhfacility/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/hhfacility
specialpurpose/jetty/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/jetty
specialpurpose/ldap/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/ldap
specialpurpose/lucene/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/lucene
specialpurpose/myportal/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/myportal
specialpurpose/pos/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/pos
specialpurpose/projectmgr/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/projectmgr
specialpurpose/oagis/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/oagis
specialpurpose/scrum/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/scrum
specialpurpose/webpos/ https://svn.apache.org/repos/asf/ofbiz/trunk/specialpurpose/webpos



Index: framework/webapp/config/url.properties
===================================================================
--- framework/webapp/config/url.properties	(revision 1608427)
+++ framework/webapp/config/url.properties	(working copy)
@@ -21,7 +21,7 @@
 ####
 
 # HTTPS Port (Secure port)
-port.https.enabled=Y
+port.https.enabled=N
 port.https=8443
 force.https.host=
 
Index: specialpurpose/build.xml
===================================================================
--- specialpurpose/build.xml	(revision 1608427)
+++ specialpurpose/build.xml	(working copy)
@@ -19,7 +19,26 @@
 -->
 
 <project name="OFBiz Special Purpose Applications Build" default="build" basedir=".">
-    <import file="../macros.xml"/>
     <filelist id="specialpurpose-builds" dir="specialpurpose"
-        files="ecommerce/build.xml"/>
+        files="
+        appserver/build.xml,
+        assetmaint/build.xml,
+        birt/build.xml,
+        ebay/build.xml,
+        ebaystore/build.xml,
+        ecommerce/build.xml,
+        example/build.xml,
+        googlebase/build.xml,
+        googlecheckout/build.xml,
+        hhfacility/build.xml,
+        jetty/build.xml,
+        ldap/build.xml,
+        lucene/build.xml,
+        oagis/build.xml,
+        pos/build.xml,
+        projectmgr/build.xml,
+        scrum/build.xml,
+        webpos/build.xml
+    "/>
 </project>
+
Index: specialpurpose/component-load.xml
===================================================================
--- specialpurpose/component-load.xml	(revision 1608427)
+++ specialpurpose/component-load.xml	(working copy)
@@ -20,5 +20,26 @@
 
 <component-loader xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="http://ofbiz.apache.org/dtds/component-loader.xsd">
+    <load-component component-location="appserver"/>
+    <load-component component-location="lucene"/>
     <load-component component-location="ecommerce"/>
+    <load-component component-location="pos"/>
+    <load-component component-location="hhfacility"/>
+    <load-component component-location="assetmaint"/>
+    <load-component component-location="cmssite"/>
+    <load-component component-location="projectmgr"/>
+    <load-component component-location="scrum"/>
+    <load-component component-location="oagis"/>
+    <load-component component-location="googlebase"/>
+    <load-component component-location="googlecheckout"/>
+    <load-component component-location="ebay"/>
+    <load-component component-location="ebaystore"/>
+    <load-component component-location="myportal"/>
+    <load-component component-location="ldap"/>
+    <load-component component-location="webpos"/>
+    <load-component component-location="birt"/>
+    <load-component component-location="example"/>
+    <load-component component-location="exampleext"/>
+    <!-- <load-component component-location="jetty"/> -->
 </component-loader>
+
